analyzer/src/main/kotlin/managers/Npm.kt

```diff
@@ -349,7 +349,7 @@ open class Npm(
 
     override fun createPackageManagerResult(projectResults: Map<File, List<ProjectAnalyzerResult>>):
             PackageManagerResult =
-        PackageManagerResult(projectResults, graphBuilder.build(), graphBuilder.packages().toSortedSet())
+        PackageManagerResult(projectResults, graphBuilder.build(), graphBuilder.packages())
```

Comment:
Nit: This formatting is inconsistent with the formatting in the changes above, and I wonder whether we should take the opportunity to reformat all of these to omit the explicit return type, as the `PackageManagerResult` is directly constructed afterwards:

    override fun createPackageManagerResult(projectResults: Map<File, List<ProjectAnalyzerResult>>) =
        PackageManagerResult(projectResults, graphBuilder.build(), graphBuilder.packages())

Reply:
Changed the formatting for all implementations of `createPackageManagerResult()` and dropped the explicit return type.